diesel_derives/src/lib.rs

```diff
@@ -78,7 +76,7 @@ use diagnostic_shim::*;
 /// `Option::<T>::None` is just skipped. To insert a `NULL` using default
 /// behavior use `Option::<Option<T>>::Some(None)`
 /// * `#[table_name = "some_table"]`, specifies the table for which the
-/// current type is a changeset. Requires that `some_table` is in scope.
```

Comment:
This remains somewhat true in my opinion. For `#[table_name = "some::releative::path"]` it is required that `some` is in scope. 
That written I'm not sure on the actual wording here.